src/coreclr/vm/hash.cpp

```diff
@@ -23,15 +23,18 @@ Module Name:
 
 //---------------------------------------------------------------------
 //  Array of primes, used by hash table to choose the number of buckets
-//  Review: would we want larger primes? e.g., for 64-bit?
 
 const DWORD g_rgPrimes[] = {
 5,11,17,23,29,37,47,59,71,89,107,131,163,197,239,293,353,431,521,631,761,919,
 1103,1327,1597,1931,2333,2801,3371,4049,4861,5839,7013,8419,10103,12143,14591,
 17519,21023,25229,30293,36353,43627,52361,62851,75431,90523, 108631, 130363,
 156437, 187751, 225307, 270371, 324449, 389357, 467237, 560689, 672827, 807403,
 968897, 1162687, 1395263, 1674319, 2009191, 2411033, 2893249, 3471899, 4166287,
-4999559, 5999471, 7199369
+4999559, 5999471, 7199369, 8639249, 10367101, 12440521, 14928637, 17914367,
+21497257, 25796711, 30956053, 37147273, 44576759, 53492113, 64190537, 77028659,
+92434393, 110921273, 133105543, 159726653, 191671993, 230006431, 276007757,
+331209331, 397451207, 476941459, 572329759, 686795723, 824154901, 988985923,
+1186783133, 1424139767, 1708967731, 2050761299
```

Comment:
Do we need to augment the list used by the managed dictionaries, too?

Reply:
I do not think so. Managed dictionaries compute the larger primes as needed. This hashtable does not have logic like that. It maxed at 7M.

This change is a quick fix. For longer term fix, we should delete this hashtable implementation (opened https://github.com/dotnet/runtime/issues/117109).

Reply:
I like the picture that this creates. It looks like the start of some really fancy 90s ASCII art!

Please add color, then the change is good to go.